### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by George7h**
In this PR the following changes were made:

- Add any missing JSX elements to your project (without adding extra functionality)
- Your final design should match Zeplin's design (this includes all elements + the same styling)
- You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/Bookform.js [hunk 1]
```diff
@@ -38,19 +38,23 @@ const BookForm = ({ onAdd }) => {
...
         type="text"
         placeholder="Author"
         value={author}
         onChange={(e) => setAuthor(e.target.value)}
       />
       <select
+        className="dropdown"
         value={category}
         onChange={(e) => setCategory(e.target.value)}
       >
```
Comment (julie-ify) [inline src/components/Bookform.js]: - [ ] Nice styling of the general app. The rest of the app meets the styling replication, but the form part can use some improvement. We are to replicate the design so the position and size of the elements should fit. The first image is your form. The second one is the design's form.

  - Kindly move the "ADD NEW BOOK" text to the same position as the design.
  - Kindly increase the length of the inputs to be just like the design. As you can see the entire length of the form stretches to the length of the grey horizontal line above it.

  image

  image

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @George7h,

Good job so far!

## Highlights!
- Good code organization and app styling ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Bookform.js (hunk 1 above)
Comment (George7h) [reply src/components/Bookform.js]: Hi @julie-ify,

Thank you for the review. I really appreciate it. I have one question... What about the Author? By not adding an input for the Author the entire book details won't be shown in the list. What do you suggest?

**Review by George7h — COMMENTED** [review]
image

**Review by DammyShittu — APPROVED** [review]
Hi @George7h,

## APPROVED 🟢

## Highlights
✔️ Project met all technical requirements

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**